### ISSUE TITLE
could we create a solarized V2 color schema with warmer or less blue light?

### ISSUE THREAD

**Issue by zhishi** [altercation/solarized#417]
I have used the solarized color schema in my emacs for several years without any complaint, and I mostly use the dark mode. That being said, I feel the dark mode is a little bit blueish. while nowadays we know that blue light is not very good for eyes and sleep especially at night, and there are different ways to reduce blue light from monitor hardware to OS settings, but that will also change the overall solarized schema to a strange feeling. 

I wonder if we could create a new version of solarized color schema with a warmer dark color, but still follow the same color contrast relationships and other attributes of original solarized schema? I know that may take lots of thoughts, test or calculation which I definitely can't do it, that's why I ask here. below is an example of warmer dark color:
![image](https://github.com/altercation/solarized/assets/7774044/6346a24a-ae1a-478c-948a-123d8613064d)

**Comment by holykremowka2137** [comment]
Do you know that blue light and blue colors are two different things, right? U can even have a pitch black screen, but without tools like redshift or gammastep (on gnome, KDE, bimbows and mac there is even built-in tool for that) your eyes will still be getting beaten up.

**Comment by slalomsk8er** [comment]
@holykremowka2137, this is pure fantasy with no basis in physics or biology:

>  U can even have a pitch black screen, but without tools like redshift or gammastep (on gnome, KDE, bimbows and mac there is even built-in tool for that) your eyes will still be getting beaten up.

I don't know how you think that the intensity of a wavelength doesn't matter. If the screen is pitch black, your eyes will not be affected at all. Also, it isn't relevant if the light comes directly form a source (light) or is reflected (color) as in the end, only the wavelengths and intensities reaching the eye are relevant and not how they were produced.

@zhishi, if I read stuff like that, one could think every person walking around in daylight will never sleep again and be blind in a month. Please tell me how the less intense light of a screen can be a problem for our eyes but the more intense natural light is fine?

Sure, you could follow the same method and select for overall warmer colors:

> I designed this colorscheme with both precise [CIELAB](http://en.wikipedia.org/wiki/Lab_color_space) lightness relationships and a refined set of hues based on fixed color wheel relationships. It has been tested extensively in real world use on color calibrated displays (as well as uncalibrated/intentionally miscalibrated displays) and in a variety of lighting conditions.

**Comment by holykremowka2137** [comment]
What? @slalomsk8er At times when i wasnt using any color correction software (reinstalling arch, doing stuff in tty or doing stuff in root account) u know what was happening? My eyes hurt like shit, but EVERY SINGLE TIME when i was greeted by beautiful red light my eyes were immediately feeling better, even when i was still using pitch black terminal emulator. 

How do you want to explain that? 

>if I read stuff like that, one could think every person walking around in daylight will never sleep again and be blind in a month. Please tell me how the less intense light of a screen can be a problem for our eyes but the more intense natural light is fine?

BECAUSE ITS NATURAL. 
Warm light is more natural for our eyes, ergo, its less harmful, our eyes literally evolved with red light. Human eyes (or any eyes at all) werent really made to cope with blue light, thats why its more harmful. 

Its literally several minutes of research with uncle google.

**Comment by slalomsk8er** [comment]
@holykremowka2137 and the sky is blue with 7 kcd/m2 compared to 80 cd/m2 | Monitor white in the sRGB reference viewing environment - I guess the natural blue daylight sky will just burn out your S cone cells.
Maybe it isn't the blue that causes you problems but the lack of red as the blue isn't gone and in the Redshift picture but reduces compared to the not schiftet version? Maybe, it's the pixels on/off ratio that changes if the mix of the sub pixel changes?
There can be a lot of reasons why that are only indirectly liked to the color itself but are caused by the display technology.
I guess in your case of apparent sensitivity to blue, I would change the color balance directly in the monitor and don't just count on color themes and color correction software.

TLDR. it isn't the wavelength that is not natural, we even have specialized cells to handle it, but the blinking of the screen is what makes it artificial and can cause all sorts of problems.
